protocol/x/clob/keeper/orders_test.go

```diff
@@ -2114,6 +2117,11 @@ func TestInitStatefulOrdersInMemClob(t *testing.T) {
 					ks.ClobKeeper.MustTriggerConditionalOrder(ks.Ctx, order.OrderId)
 				}
 
+				// Increment the expected count for non-conditional orders and triggered conditional orders.
```

Comment:
Shouldn't the number of open stateful orders include untriggered conditional orders in this test expectation?

Reply:
No, `GetStatefulOrderCount` represents all triggered and non-conditional stateful orders. Untriggered orders are counted separately.

Minor update to comment.